test/test_easteregg.py

```diff
@@ -0,0 +1,12 @@
+from testing_base import *
```

Comment:
This test looks good. Covers basically both of the two possible options for this function in gameplay.